x-pack/plugins/observability_solution/entity_manager/server/lib/entities/transform/generate_history_transform.ts

```diff
@@ -26,12 +26,37 @@ import {
 import { isBackfillEnabled } from '../helpers/is_backfill_enabled';
 
 export function generateHistoryTransform(
```

Comment:
split this function in two as I found the backfill conditionals + fallbacks made it hard to follow